test/schema_abort_predictable.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Get a random value with leading zeroes removed, /bin/sh version.
+rando() {
+  tr -cd 0-9 </dev/urandom | head -c 5 | sed -e 's/0*\(.\)/\1/'
+}
+
+if [ $# -lt 1 ]; then
+    echo "Error: Not enough arguments given."
+    echo "Current args: $@"
+    exit 1
+fi
+
+times=$1
+
+runtime=20  # seconds
+nthreads=5
+
+toolsdir=../../../../tools
```

Comment:
These hardcoded paths assume the script is run from `wiredtiger/cmake_build/test/csuite/schema_abort` which we set with `working_dir` in the evergreen task. 
Now this code is in a script that requirement is less obvious. Can we have a naiive check:
```
if [ "basename $(pwd)" != "schema_abort" ];
    echo "This script must be run from the build/test/csuite/schema_abort folder"
    exit 1;
fi
```
?

Same comment for the hardcoded paths in `checkpoint_test_predictable`

Reply:
check added in [6299b97](https://github.com/wiredtiger/wiredtiger/pull/10921/commits/6299b97bfd7d576aa51b350f8eede428784cd1f5)